### DISCUSSION TITLE
disable 'c-/' causes error

### DISCUSSION THREAD

**Discussion by ssh352** [LazyVim/LazyVim#3032] [Q&A] [answered]
I'd like to disable c-/ and I have this in my keymaps.lua. However it generates an error `E31: no such mapping` Please help, thanks!
```
vim.keymap.del("n", "c-/")
```

the default keymaps.lua
```
 -- floating terminal
 local lazyterm = function() LazyVim.terminal(nil, { cwd = LazyVim.root() }) end Unresolved lens ... 3 references
 map("n", "<leader>ft", lazyterm, { desc = "Terminal (Root Dir)" })
 map("n", "<leader>fT", function() LazyVim.terminal() end, { desc = "Terminal (cwd)" })
 map("n", "<c-/>", lazyterm, { desc = "Terminal (Root Dir)" })
 map("n", "<c-_>", lazyterm, { desc = "which_key_ignore" })
```

**Comment by dpetka2001** [comment, marked as answer]
First of all your syntax is wrong. It should be `vim.keymap.del("n", "<c-/>")`. Second, you should know that some terminals interpret `<C-/>` as `<C-_>`. To verify how your terminal interprets `<C-/>` inside Neovim, enter insert mode by pressing `i` and then press `<C-v><C-/>` and note what characters are printed out. Those characters you should use in your keymap.
